pyensembl/gtf_parsing.py

```diff
@@ -121,20 +122,30 @@ def _extend_with_attributes(df):
     extra_columns = {}
     column_order = []
     for i, attr_string in enumerate(df.attribute):
+        # TODO: tokenize attribute strings properly
+        # for now, catch mistaken semicolons by replacing "xyz;" with "xyz"
+        # Required to do this since the GTF for Ensembl 78 has
+        # gene_name = "PRAMEF6;"
+        # transcript_name = "PRAMEF6;-201"
+        attr_string = attr_string.replace(';\"', '\"').replace(";-", "-")
+
         pairs = (
             kv.strip().split(" ", 2)
             for kv in attr_string.split(";")
-            if len(kv) > 0
+            # simplest entry must be at least three characters:
+            # (key name, space, value)
```

Comment:
KV pairs are split by semicolons, the key and value themselves are split by a space. 

Example:

    gene_name "WOLFDOG"; transcript_name "WOLFDOG-001"

In the malformed GTF file I got for release 78, there were entries like:

    gene_name "WOLFDOG;"; transcript_name "WOLFDOG;-001"